src/com/owncloud/android/lib/resources/files/model/RemoteFile.java

```diff
@@ -232,10 +232,13 @@ public RemoteFile(WebdavEntry we) {
         setRemoteId(we.remoteId());
         setSize(we.size());
         setFavorite(we.isFavorite());
+        setIsEncrypted(we.isEncrypted());
```

Comment:
Only folders are marked as encrypted, thus this is only in ReadFolderRemoteOperation:
https://github.com/nextcloud/android-library/blob/56cdc0cf218af7104ace78f4c69e6072d2cd502f/src/com/owncloud/android/lib/resources/files/ReadFolderRemoteOperation.java#L175

So please remove this.

Reply:
Ergh, but a RemoteFile could be a folder too?

Reply:
Yes, but this is not used in FolderOperation and for files we do not need it as it is set by the parent folder.
So please remove this for now, and if we put all the different places together, we need to make sure that it will still work.

Reply:
Don't understand the reasoning I must admit? Like I told you, you can use ReadFileRemoteOperation to get folder information WITHOUT descending into all the children.

So this is indeed helpful to me.

Reply:
Encryption API is not setting encryption flag on file base, but only on folders, as you cannot have an encrypted file without an encrypted parent folder. 
So this is just not needed. If you really want to have it in, please add an check if it is a folder.